opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -661,7 +662,7 @@ class Well {
 
     Status status;
     PAvg m_pavg;
-    double well_temperature;
+    std::optional<double> well_temperature;
```

Comment:
Will this temperature ever be related to producers? If not, maybe we should change the variable name to be inj_temperature, something like that so we make sure it is only related to injectors.